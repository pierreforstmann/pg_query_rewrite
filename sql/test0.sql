drop extension if exists pg_query_rewrite;
--
create extension pg_query_rewrite;
select pgqr_truncate_rule();
--
select pgqr_add_rule('select 10;','select 11;');
--
select 10;
--
drop extension pg_query_rewrite;
