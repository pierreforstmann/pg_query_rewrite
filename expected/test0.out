drop extension if exists pg_query_rewrite;
NOTICE:  extension "pg_query_rewrite" does not exist, skipping
--
create extension pg_query_rewrite;
--
select pgqr_add_rule('select 10;','select 11;');
 pgqr_add_rule 
---------------
 t
(1 row)

--
select 10;
 ?column? 
----------
       11
(1 row)

--
drop extension pg_query_rewrite;
